tests/test_time_utils.py

```diff
@@ -0,0 +1,30 @@
+# This test is written by Andrew Ng, 19/03/22. It makes use of several CDMs provided by the NASA CARA
```

Comment:
Once you move the content of `time_utils.py` inside `util.py`, we should also move this test inside the `test_util.py` file.